### PR TITLE
Update itemstats.js

### PR DESCRIPTION
**Pull request by DeaJae**
Again, because i suck at pull requests.

### CODE REVIEW

File: src/lib/itemstats.js [hunk 1]
```diff
@@ -76,7 +76,8 @@ function getCalcFunction(itemAttributes, selectedStat) {
       return getCalcFunctionStandard(itemAttributes, selectedStat);
     case 4:
       return getCalcFunctionQuad(itemAttributes, selectedStat);
-    case 7:
+    // case 7:
```
Comment (madou) [inline src/lib/itemstats.js:79]:
```suggestion
```

**Comment by itsdouges** [comment]
Thanks I'll deploy this when I can. Can you give an example request where it should work?

**Comment by itsdouges** [comment]
I've deployed both read and bulk_read. Let me know if it's fixed for you.

**Comment by DeaJae** [comment]
Just deployed a test template build, it works great.
celestial fixed


Thanks

**Comment by itsdouges** [comment]
Lovely no worries, swing back if anything else needs updating.

**Pull request merged**